### PR TITLE
Add shortened common onvif namespace prefixes

### PR DESCRIPTION
**Pull request by jterrace**
This fixes some Tapo cameras that respond with a 500 error when receiving the ns0/ns1 namespace prefix with embedded xmlns definitions inside each element.

A similar change was done upstream here:
https://github.com/FalkTannhaeuser/python-onvif-zeep/pull/46

for https://github.com/mvantellingen/python-zeep/issues/870.

This fixes the main barrier for getting https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/304 fixed.

I also updated the events example that was broken.

### CODE REVIEW

**Comment by bdraco** [comment]
I can take a look the week after Thanksgiving

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Package
+ciso8601>=2.1.3
```
Comment (bdraco) [inline requirements.txt]:
```suggestion
ciso8601==2.1.3
```

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
Tested with a few random ONVIF cameras and didn't observe any regressions

**Comment by jterrace** [comment]
Thanks, Nick!

**Comment by bdraco** [comment]
[3.1.13](https://github.com/openvideolibs/python-onvif-zeep-async/releases/tag/v3.1.13) published

**Comment by bdraco** [comment]
I didn't do a dep update PR to home-assistant/core as I ran out of time getting after getting the CI/codecov/publishing working on this repo.  If someone could do that, it would be appreciated.

**Comment by jterrace** [comment]
I'm going to update https://github.com/home-assistant/core/pull/130769 to include the version bump.

**Comment by bdraco** [comment]
Please follow HA best practices and do a separate PR for the version bump

https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=smalle#5-make-your-pull-request-as-small-as-possible

**Comment by jterrace** [comment]
Okay, will leave it alone then and submit a separate PR

**Comment by bdraco** [comment]
Example of previous dep bumps for various integrations https://github.com/home-assistant/core/pulls?q=sort%3Aupdated-desc+is%3Apr+label%3Adependency+is%3Aclosed

**Pull request merged**